subhub/customer.py

```diff
@@ -20,6 +21,10 @@ def create_customer(
 ) -> Customer:
     # First search Stripe to ensure we don't have an unlinked Stripe record
     # already in Stripe
+    if not origin_system.lower() in CFG.ALLOWED_ORIGIN_SYSTEMS:
+        raise InvalidRequestError(
+            message="Invalid origin_system provided", param=str(origin_system)
```

Comment:
the ALLOWED_ORIGIN_SYSTEMS are _not_ being passed to the Exception as we discussed.